xla/ffi/api/api.h

```diff
@@ -1628,7 +1628,7 @@ class Handler : public Ffi {
                                                                       \
       return *reinterpret_cast<T*>(scalar->value);                    \
     }                                                                 \
-  }
+  };
```

Comment:
`;` intentionally omitted here, as macro itself should end with `;`

Reply:
fixed